.github/workflows/pr-request-review.yml

```diff
@@ -0,0 +1,33 @@
+name: Request Review
+
+on:
+  issue_comment:
+    types:
+      - created
+
+jobs:
+  request-review:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    if: github.repository_owner == 'cryostatio' && github.event.issue.pull_request && startsWith(github.event.comment.body, '/request_review')
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        
+      - name: Add label and request review
+        run: |
+          reviewers="cryostatio/reviewers"
+
+          echo "Adding label 'review-requested' to the PR"
+          gh pr edit ${{ github.event.issue.number }} --add-label "review-requested"
+
+          if [ -n "$reviewers" ]; then
```

Comment:
What is this `if` statement needed for? It looks like the `reviewers` variable is declared above and always has the value `cryostatio/reviewers`

Reply:
it checks if specified reviewers were mentioned if not defaults to the team

Reply:
I don't understand how it does that... where is the "specified reviewers" check done?

Reply:
I see in the other commend thread there is a `sed` command that looks like it will do some extraction like that, but it isn't committed perhaps?

Reply:
with the -n flag it checks that the `reviewers` variable its not empty. if not empty assumes that one has explicitly specified reviewers and adds it to the reviewers?

Reply:
I think it will never be empty because on line 20, the value is hardcoded to `cryostatio/reviewers` within the script

Reply:
> I think it will never be empty because on line 20, the value is hardcoded to `cryostatio/reviewers` within the script

Yes, that is correct.

Reply:
I don't understand then. If the variable is always hardcoded to that value, there is no "if there was a reviewer specified" and this `if` will always take the one side of the branch

Reply:
[https://github.com/aali309/cryostat/actions](https://github.com/aali309/cryostat/actions)
looks like it is working with the suggested change. Im wondering with the testing I used individual accounts to send requests using the flag `--add-reviewer`. I thought there was a flag `--team` since the `cryostat/reviewers` is a team, but I dint see it in all the resources online.  will just `--add-reviewer $reviewers`  work the way I pushed it or do i need to add a for loop like. @andrewazores please let me know
```    echo "Requesting review from @cryostatio/reviewers team"
          for reviewer in $reviewers; do
            gh pr edit ${{ github.event.issue.number }} --add-reviewer $reviewers
          done```

Reply:
I think you can just treat the "reviewers" as if it is a single person's account. When such a team is requested as the reviewer then the GitHub site/platform will add it as if it's a single account and notify everyone who is a member of that team.

Reply:
Thank you @andrewazores  for the clarification. With that being said, I think all works well now.